internal/types/studies.go

```diff
@@ -6,9 +6,34 @@ import (
 
 type Study struct {
 	ModelAuditable
-	OwnerUserID uuid.UUID `gorm:"not null;index"`
-	Title       string    `gorm:"not null"`
-	Description string    `gorm:"type:text"`
+	OwnerUserID                      uuid.UUID `gorm:"not null;index"`
+	Title                            string    `gorm:"not null"`
+	Description                      *string   `gorm:"type:text"`
+	Admin                            *string   `gorm:"type:varchar(255)"`
+	Controller                       string    `gorm:"not null"`
+	ControllerOther                  *string   `gorm:"type:varchar(255)"`
```

Comment:
I'm not a big fan of this. querying will be more complicated than it needs to be(?). all of `ucl`, `Ucl` and `UCL` `University college london` should be stored as the same thing